spec/specialist_publisher/change_note_spec.rb

```diff
@@ -54,7 +54,7 @@ def then_i_can_view_the_change_note_on_gov_uk
 
     click_link("View on website")
     expect_url_matches_live_gov_uk
-    click_link("+ full page history")
+    click_link("+ show all updates")
```

Comment:
There's an opportunity here to move to html entities in [the published-dates component](https://github.com/alphagov/government-frontend/blob/master/app/views/components/_published-dates.html.erb#L18)

The published dates component was originally written using html entities:
https://github.com/alphagov/government-frontend/commit/9297f2d438216491901e441391f6cf37d5a72690#diff-4b66b1be92a71f30b589d047d897a003

Reply:
The plus symbol was already generated by an html entity in the PR. I've updated so that the minus symbol is too. I think the capybara selector will only work as it stands here, though - it reads what the front end renders, not the symbol that generates it, as per https://github.com/teamcapybara/capybara/issues/1341